administration/src/components/verification/VerificationController.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button, Card, H1, Spinner } from '@blueprintjs/core'
+import React, { useCallback, useContext } from 'react'
+import { useLocation, useParams } from 'react-router-dom'
+import styled from 'styled-components'
+import JsonFieldView from '../applications/JsonFieldView'
+import ErrorHandler from '../../ErrorHandler'
+import { useGetApplicationQuery, useVerifyApplicationVerificationMutation } from '../../generated/graphql'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import NotFoundView from '../NotFoundView'
+import { useAppToaster } from '../AppToaster'
+
+const CARD_PADDING = 20
+
+const ApplicationViewCard = styled(Card)`
+  width: 800px;
+  margin: 10px;
+`
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  justify-content: center;
+  padding: ${CARD_PADDING}px;
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  width: inherit;
+  flex-direction: row;
+  justify-content: space-around;
+`
+
+type ApplicationVerificationProps = {
+  applicationVerificationId: number
+  applicationAccessKey: string
+}
+
+const ApplicationVerification = ({ applicationVerificationId, applicationAccessKey }: ApplicationVerificationProps) => {
+  const config = useContext(ProjectConfigContext)
+  const appToaster = useAppToaster()
+  const showErrorToaster = () => appToaster?.show({ intent: 'danger', message: 'Etwas ist schief gelaufen.' })
+  const [verifyApplicationVerification, { loading: verificationLoading }] = useVerifyApplicationVerificationMutation({
+    onError: error => {
+      console.error(error)
+      showErrorToaster()
+    },
+  })
+
+  const submitApplicationVerification = useCallback(
+    (verified: boolean) => {
+      verifyApplicationVerification({
+        variables: {
+          applicationVerificationId,
+          verified,
+        },
+      })
+    },
+    [applicationVerificationId, verifyApplicationVerification]
+  )
+
+  const { loading, error, data, refetch } = useGetApplicationQuery({
+    variables: { applicationVerificationId, accessKey: applicationAccessKey },
+  })
+
+  if (loading) return <Spinner />
+  else if (error || !data) return <ErrorHandler refetch={refetch} />
+
+  const { jsonValue, id, verifications } = data.application
+  const verification = verifications.length >= 0 ? verifications[0] : null
```

Comment:
```suggestion
  const verification = verifications.length > 0 ? verifications[0] : null
```
i think the length shouldn't be 0, no? I even dunno why it can have multiple verifacations? :(

Reply:
Jep. it is possible and a typo as well. Further, I now added, that the list is filtered for the organisation, that opened the link to make it more clear.

Reply:
@f1sh1918 : In the Antrag, you can specify multiple organizations that you have been volunteering at (over the last years). That's why multiple organizations can verify the application.

Reply:
ok but if we have multiple organization, every organization has to receive an email and you have to specify the organization by some identifier maybe also in the link and resolve it here instead of using `verifications[0]`

Reply:
this snippet is already outdated. see the updated code :)